<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActivitiesForm_template</class>
 <widget class="QDialog" name="ActivitiesForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>514</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string extracomment="This is the title of the activities dialog">Activities dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="activeTextLabel">
           <property name="text">
            <string notr="true">No: 1000 / 1000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="totalTextLabel">
           <property name="text">
            <string notr="true">Dur: 1000 / 1000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QListWidget" name="activitiesListWidget">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
            </font>
           </property>
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="clearPushButton">
         <property name="text">
          <string>Remove All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filterGroupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>160</height>
          </size>
         </property>
         <property name="title">
          <string>Filter</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="studentsTextLabel">
            <property name="text">
             <string comment="It refers to students sets, but I want to keep the string short">Students</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="studentsComboBox">
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="recursiveCheckBox">
            <property name="text">
             <string>Show related</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="subjectsTextLabel">
            <property name="text">
             <string>Subject</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="subjectsComboBox">
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="activityTagsTextLabel">
            <property name="text">
             <string>Activity tag</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="activityTagsComboBox">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Current activity</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="activityTextEdit">
         <property name="font">
          <font>
           <family>Ubuntu Mono</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="modifyActivityPushButton">
           <property name="text">
            <string>Modify</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="commentsPushButton">
           <property name="text">
            <string>Comments</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeActivityPushButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>30</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addActivityPushButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="helpPushButton">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
